go/vt/vttablet/tabletserver/query_engine.go

```diff
@@ -56,6 +56,7 @@ import (
 type TabletPlan struct {
 	*planbuilder.Plan
 	Original   string
+	Statement  sqlparser.Statement
```

Comment:
I am a little unsure about adding the Statement here too since we have the query in *planbuilder.Plan

Reply:
I originally tried to use the `FullStmt` available in `*planbuilder.Plan`. Ran into two issues:

 * Many of the `Plan` produced by planbuilder don't include `FullStmt`, including `SELECT` plans.
 * I tried modifying the `planbuilder` code to always include `FullStmt`, but that broken some of the tests we have around plan caching, and I was hesitant about making any changes there.